### PR TITLE
Update Module Name to support WMF 5

### PR DESCRIPTION
**Pull request by TravisEz13**
Update Module Name to support WMF 5

WMF 5 doesn't support resources being named the same as a module.

### CODE REVIEW

File: DscResources/xPhpProvision/xPhpProvision.Schema.psm1 [hunk 1]
```diff
@@ -1,134 +1,143 @@
...
-            Name = "Path"
-            Value = ";$($DestinationPath)"
-            Ensure = "Present"
-            Path = $true
-            DependsOn = "[Archive]PHP"
-        }
-}
-
-
+# Composite configuration to install the IIS pre-requisites for php
```
Comment (TravisEz13) [inline DscResources/xPhpProvision/xPhpProvision.Schema.psm1:1]: Line endings were wrong and/or encoding, I forget

File: DscResources/xPhpProvision/xPhpProvision.psd1 [hunk 2]
```diff
@@ -1,96 +1,94 @@
...
-# HelpInfoURI = ''
-
-# Default prefix for commands exported from this module. Override the default prefix using Import-Module -Prefix.
-# DefaultCommandPrefix = ''
-
-}
-
-
-
+#
```
Comment (TravisEz13) [inline DscResources/xPhpProvision/xPhpProvision.psd1:1]: line-endings were wrong or encoding, I forget.

File: README.md [hunk 3]
```diff
@@ -26,6 +26,11 @@ Please check out common DSC Resources [contributing guidelines](https://github.c
 * Initial release with the following resources 
     - xPhp
 
+### 1.0.?
```
Comment (TravisEz13) [inline README.md]: Not sure want to put for the vNext version?

Comment (joeyaiello) [inline README.md]: Hmmm... @vors seems like kind of a chicken/egg problem. Should we generate this from merged pull requests and add it to the changelog when we merge dev to master and rev the version?

File: DscResources/xPhpProvision/xPhpProvision.Schema.psm1 [hunk 4]
```diff
@@ -1,134 +1,143 @@
...
+Configuration IisPreReqs_php
+{
+param
```
Comment (xiaoyongaz) [inline DscResources/xPhpProvision/xPhpProvision.Schema.psm1]: [nit] space?

File: DscResources/xPhpProvision/xPhpProvision.Schema.psm1 [hunk 5]
```diff
@@ -1,134 +1,143 @@
...
+        [switch] $installMySqlExt,
+
+        [Parameter(Mandatory = $true)]
+        [string] $PackageFolder,
+
+        [Parameter(Mandatory = $true)]
+        [string] $DownloadUri,
+
+        [Parameter(Mandatory = $true)]
+        [string] $Vc2012RedistDownloadUri,
```
Comment (xiaoyongaz) [inline DscResources/xPhpProvision/xPhpProvision.Schema.psm1]: should we have some default values for the uris and destinations?

File: DscResources/xPhpProvision/xPhpProvision.Schema.psm1 [hunk 6]
```diff
@@ -1,134 +1,143 @@
...
+        {
+            $phpZip = $DownloadURI
+        }
+
+        # Make sure the content of the PHP archine are in the PHP path
+        Archive php
+        {
+            Path         = $phpZip
+            Destination  = $DestinationPath
+            DependsOn    = $archiveDependsOn
```
Comment (xiaoyongaz) [inline DscResources/xPhpProvision/xPhpProvision.Schema.psm1:89]: nice way to express this dependson

**Comment by xiaoyongaz** [comment]
overall looks fine too me, comments can be addressed later

File: README.md (hunk 3 above)
Comment (vors) [inline README.md]: Pester use 'unreleased' version for that. I don't think picking the name make difference. Call it vnext or unreleased or ???.

File: DscResources/xPhpProvision/xPhpProvision.Schema.psm1 (hunk 5 above)
Comment (TravisEz13) [inline DscResources/xPhpProvision/xPhpProvision.Schema.psm1]: The php url changes frequently, so it should stay data.
Updated the destination parameters to have defaults.

**Comment by xiaoyongaz** [comment]
looks good

**Pull request merged**